### ISSUE TITLE
Add description/instructions as to what user should do when they land on the front form page.

### ISSUE THREAD

**Issue by DRD161** [marleymarl/geotime#13]
We should probably add some sort of disclaimer or description to the top of UpFrontForm component to tell users where they are, what the call to action is, etc. I feel this will make for a better user experience.

**Comment by marleymarl** [comment]
Agreed. There's a couple of components to this though. First UpFrontForm needs to be wrapped in a GeoTimeline component (that wraps both MapContainer and UpFrontForm) and then description, disclaimer etc. could be passed in as props. Someone needs to style the UpFrontForm and take in those props and put them where they should go. So I think this should a separate issue. This issue applies more to our own use of the library on the demo site (i.e. the form page on the demo site).

**Comment by DRD161** [comment]
Couldn't we just add the text to the form component itself? Like a header and some text? Or did you have a different idea?

**Comment by marleymarl** [comment]
We could, but then that info would be 'hard-coded' in the form as is. If we pass it in as a prop then others could change the text etc. by changing the prop. It depends if we want to include the form in the package or just as a standalone for the demo only. I'm leaning towards putting it in.

**Comment by DRD161** [comment]
Ahh ok I see what you mean. That makes total sense. That way when other developers use our app in their own apps they will just have to change the props to change the description to their liking. I can dig it :)

**Comment by DRD161** [comment]
I'll tackle this. I'll add additional props to the component that accept the description text as well as an image. Other developers may want to use different background images.

**Comment by DRD161** [comment]
Should we still hold off on this until all the ui/ux task are completed? I can add the props to the component with placeholder text or add the props and comment them out for when we need them. Thoughts?

**Comment by heldersepu** [comment]
@DRD161 anything you see of value add it
... we can polish later (and I :heart: placeholders)

**Comment by DRD161** [comment]
@heldersepu haha will do! I only ask because in the beginning I made some changes before things were more ironed out and we had to revert. No worries though I’m on it 👍

**Comment by heldersepu** [comment]
That's why we use git... if all life had a revert option we would not be on this Covit19 mess

**Comment by DRD161** [comment]
This has been completed and merged.

**Comment by DRD161** [comment]
This has been completed and merged.

**Comment by DRD161** [comment]
Just placeholders for now. As I mentioned in the commit message until we decide on an actual description we can comment the props out. I didn't want to add a description without it being somewhat of a group decision. I can add in any header and description text we come up with when the time comes.

**Comment by heldersepu** [comment]
in that case, let's keep this open for someone to add a proper description

**Comment by DRD161** [comment]
Seems like a proper title and description was added to the app.geotimeline site. We should use this to keep things consistent.

<img width="1645" alt="Screen Shot 2020-04-07 at 8 56 19 AM" src="https://user-images.githubusercontent.com/29187448/78672478-b1bfc880-78ae-11ea-9517-e3faa9f997bb.png">

**Comment by marleymarl** [comment]
Agreed. I was trying to sneak another feature in before pushing but deployed the description to the site before I went to sleep for a bit. My bad - will tidy of the feature and push soon to update. Should have pushed before starting the other feature. :)

**Comment by DRD161** [comment]
Haha no worries! Should I go ahead and add the text shown in the screenshot above to my branch and submit a PR?

**Comment by marleymarl** [comment]
Haha sure. :)

**Comment by marleymarl** [comment]
Actually nvm I'm going to push what I have in a sec.

**Comment by DRD161** [comment]
Ok no worries. I'll add the text to the props later on and submit a PR for the next time you push.

**Comment by marleymarl** [comment]
I've already got it added to the props.

**Comment by DRD161** [comment]
Oh ok perfect! 👍

**Comment by DRD161** [comment]
Proper title and description has been added and pushed. Closing this issue.
